cli.js

```diff
@@ -158,7 +158,10 @@ function exit(results) {
 	// correctly flush the output when multiple test files
 	process.stdout.write('');
```

Comment:
Yeah, I added it as a quick hack because the `1 test passed` message wouldn't output without it when running multiple tests and I couldn't reproduce it in a test. If it works fine now you can just remove it.